Makefile.am

```diff
@@ -6694,6 +6698,21 @@ tests += \
 	test-libsystemd-sym \
 	test-libudev-sym
 
+.PHONY: install-tests
+install-tests: $(tests) $(TEST_DATA_FILES)
```

Comment:
Hm, that's problematic.

We have `--enable-tests=unsafe`. I think it's reasonable to enable this during package build (since it usually happens in a throw-away environment), but then those tests are installed intermingled with the other ones. Also, the result of install-tests depends on this configure option, but I don't think it should.

Maybe the extra tests should be installed into `$(testsdir)/{unsafe,manual}`?